models/src/main/java/com/vimeo/networking2/Album.kt

```diff
@@ -0,0 +1,119 @@
+package com.vimeo.networking2
+
+import com.vimeo.networking2.enums.AlbumLayoutType
+import com.vimeo.networking2.enums.AlbumThemeType
+import com.vimeo.networking2.enums.SortType
+import java.util.*
+
+/**
+ * Group of videos to share publicly or privately.
+ */
+data class Album(
+
+    /**
+     * Hexadecimal color code for the decorative color.
+     * For example, album videos use this color for player buttons.
+     */
+    val brandColor: String? = null,
+
+    /**
+     * The time in ISO 8601 format that the album was created.
+     */
+    val createdTime: Date? = null,
+
+    /**
+     * Custom logo.
+     */
+    val customLogo: PictureCollection? = null,
+
+    /**
+     * A brief description of the album's content.
+     */
+    val description: String? = null,
+
+    /**
+     * The total duration in seconds of all the videos in the album.
+     */
+    val duration: Int? = null,
+
+    /**
+     * Embed data for the album.
+     */
+    val embed: AlbumEmbed? = null,
+
+    /**
+     * Whether to hide the Vimeo navigation when viewing the album.
+     */
+    val hideNav: Boolean? = null,
+
+    /**
+     * The album's layout preference.
+     */
+    val layout: AlbumLayoutType? = null,
+
+    /**
+     * The URL to access the album.
+     */
+    val link: String? = null,
+
+    /**
+     * Metadata about the album.
+     */
+    val metadata: Metadata<AlbumConnections, AlbumInteractions>? = null,
+
+    /**
+     * The time in ISO 8601 format when the album was last modified.
+     */
+    val modifiedTime: Date? = null,
+
+    /**
+     * The album's display name.
+     */
+    val name: String? = null,
+
+    /**
+     * The active image for the album; defaults to the thumbnail of the last
+     * video added to the album.
+     *
+     * This data is given only by versions above 3.4.1.
+     */
+    val albumPicture: PictureCollection,
+
+    /**
+     * List of the first three thumbnails of videos in the album.
+     *
+     * This data is given only by version 3.4.1 of the API.
```

Comment:
Since we're locking the API version of each release of the networking library (and not letting consumers specify), this _might_ not be necessary since every model will always represent the current response from the server.

But this _could_ be necessary for supporting locally cached objects (if we choose to do so). If that's the case, we should consider GSON as an option, since it allows for version annotations on fields and we could just use that to specify when a field was added (based on the API version, not library version).

Either way a ticket to follow up on this might be good if one doesn't exist.

Reply:
I will add a separate ticket for specifying fields that are only available through certain API versions.